apps/prairielearn/src/lib/cache.ts

```diff
@@ -41,7 +41,7 @@ export async function init() {
   }
 }
 
-export function set(key: string, value: any, maxAgeMS?: number) {
+export function set(key: string, value: any, maxAgeMS: number) {
```

Comment:
We could remove the `?? undefined` when we use `maxAgeMS` below?

Also lol about the comment that we don't care whether it errors.

Reply:
😭 I know, we should really improve that, at the very least an OTel metric (these are way too high-volume to report to Sentry).

I'll save that for the future though.

Reply:
Removed `?? undefined` in 9319a08 (I misstyped "fallbacks" as "callbacks" in the commit message lol)